frontend2/src/views/QuickStart.tsx

```diff
@@ -1,7 +1,9 @@
 import React from "react";
+import DocumentationPage from "../components/DocumentationPage";
+import { BC23_QUICKSTART } from "../content/bc23";
 
 const QuickStart: React.FC = () => {
```

Comment:
Is this view going to always bring up `bc23`'s Quickstart? Or are we planning to use EpisodeContext on this page (e.g. `bc22`,`bc23`) that indexes and retrieves `<episodeId>_QUICKSTART` for the `Quickstart` component?

Reply:
yup, this will be a later change where we'll get the episode id and display that episode's quickstart